src/components/opt-in-flyout/flyout-impl.js

```diff
@@ -47,13 +47,19 @@ class FlyoutImplementation extends composeMixins(
 			heading1Styles,
 			css`
 				:host {
-					height: var(--d2l-flyout-custom-element-height, 100%);
+					height: var(--d2l-flyout-custom-element-height, auto);
 					overflow: hidden;
 					pointer-events: none;
 					position: absolute;
 					width: 100%;
 				}
 
+				:host([opened]) {
+					background-color: rgba(255, 255, 255, 0.7);
```

Comment:
Curious what the semi-transparency is being used for here.

Reply:
I see what this is being used for now. Follow-up question, do you know why this is using `white` as opposed to regolith like [d2l-backdrop](https://github.com/BrightspaceUI/core/blob/main/components/backdrop/backdrop.js#L46). I see the transparency is at least consistent.